### DISCUSSION TITLE
Trimming whitespace in between and around union types

### DISCUSSION THREAD

**Discussion by finn-matti** [squizlabs/PHP_CodeSniffer#3486] [Q&A] [answered]
If I have a parameter signature like this:

```
public function foo (bar   | foobar     |baz  $item)
```

Can php_codesniffer trim the whitespace to make this look fine?

```
public function foo (bar|foobar|baz $item)
```

How would I need to configure phpcs/phpcbf for this?

**Comment by gsherwood** [comment, marked as answer]
There is no included sniff that enforces the rule you are asking about.

**Reply by finn-matti** [reply]
Ah, I see. What a pity...
